### ISSUE TITLE
Map provider in training and IG

### ISSUE THREAD

**Issue by agpul** [cyfronet-fid/eosc-search-service#775]
AC:
 - [ ] As a User, I would like to see providers mapped to proper values (as a reference see Services) for Providers and Trainings

TLDR:
In guidelines (and probably training as well) "provider" property is a pid. 
The scope of the ticket is to: request provider solr collection using the pid value, and take the title (please test whether is should be a title - I am guessing) of the given provider. 

Places that I see bad mapped values:
- Trainings collection:
![image](https://github.com/cyfronet-fid/eosc-search-service/assets/43000481/eb9a187f-599a-4941-8273-9a7b76d87419)

- Training detail page (two fields):
![image](https://github.com/cyfronet-fid/eosc-search-service/assets/43000481/b45de2fe-0be8-4a69-9e99-369f311549db)

- Interoperability Guidelines collection:
![image](https://github.com/cyfronet-fid/eosc-search-service/assets/43000481/c293385d-74fe-4895-a2ff-ee03073685fb)

- Interoperability Guidelines detail page:
![image](https://github.com/cyfronet-fid/eosc-search-service/assets/43000481/695e9d58-95dd-4ce9-89c1-299da0376371)

Figma:

**Comment by agpul** [comment]
Estimation: 1 day

**Comment by maria-j-k** [comment]
blocked by #857
